cmdk/src/index.tsx

```diff
@@ -576,7 +576,13 @@ const Command = React.forwardRef<HTMLDivElement, CommandProps>((props, forwarded
       onKeyDown={(e) => {
         etc.onKeyDown?.(e)
 
-        if (!e.defaultPrevented) {
+        // Check if IME composition is finished before triggering key binds
+        // This prevents unwanted triggering while user is still inputting text with IME
+        // e.keyCode === 229 is for the CJK IME with Legacy Browser [https://w3c.github.io/uievents/#determine-keydown-keyup-keyCode]
+        // isComposing is for the CJK IME with Modern Browser [https://developer.mozilla.org/en-US/docs/Web/API/CompositionEvent/isComposing]
+        const isComposing = e.nativeEvent.isComposing || e.keyCode === 229
+
+        if (!e.defaultPrevented || !isComposing) {
```

Comment:
Shouldn't this be && ?
```suggestion
        if (!e.defaultPrevented && !isComposing) {
```